_posts/2021-01-22-reaching-out.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "2021-01-22: Reaching the IAC Team"
+author: pmcmaw
+categories:
+  - team
+tags:
+  - community
+  - reviews
+  - modules
+  - windows
+  - communication
+---
+## Reaching the IAC Team ![hello]({% link /assets/2021-01-22-reaching-out/hello.png %}){:height="40px" width="40px"}
+
+To improve collaboration and avoid issues falling through the cracks, we have collected information on all the ways to get in contact with us. Whether this be to review a PR, highlight a nasty bug or really for any other reason, let us know what we did well, and what we need to do better.
+
+
+## Monday is Community Day ![slack]({% link /assets/2021-01-22-reaching-out/calendar.png %}){:height="30px" width="30px"}
+
+Yes, that's right! The whole team dedicates a full day to you every week. Every Monday we go over pull requests, releases, new issues and anything else module related.
+
+
+## Reaching out via Slack ![slack]({% link /assets/2021-01-22-reaching-out/slack.png %}){:height="45px" width="45px"}
+
+As part of Community Monday, we have weekly office hours on Monday 1.00pm and 5.00pm GMT alternating each week. Check the [calendar](https://puppet.com/community/office-hours) for the events in your timezone. Join #office-hours on our [community-slack](https://puppetcommunity.slack.com). Between these hours the IAC team will be online and can chat about anything module related and much more!
+
+Outside the office hours, some of us hang out in #forge-modules and #testing on our [community-slack](https://puppetcommunity.slack.com). Even if nobody from the team is around, other community members might be able to help.
+
+**Use the ![sos]({% link /assets/2021-01-22-reaching-out/sos.gif %}){:height="50px" width="50px"}
+ emoji on urgent issues that require the attention of a puppet employee.**
+
+## Reaching out via Jira ![bug]({% link /assets/2021-01-22-reaching-out/bug.png %}){:height="40px" width="40px"}
+
+When you come across an issue with one of our supported modules and want to raise an issue, in order for to it to come directly to us there are a few things we would doing first.
+
+1) Confirm this issue hasn't already been raised by using the JIRA search function
+
+2) If the repository has issues enabled, use this instead of JIRA
+
+Once the above has been confirmed, it is recommended that you complete the following settings in the ticket. This will ensure that our team sees the ticket pretty quick.
```

Comment:
```suggestion
Once the above has been confirmed, it is recommended that you complete the following parameters in the ticket. This will ensure that our team sees the ticket pretty quick.
```